WordPress/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
         minSdkVersion 14
         targetSdkVersion 23
 
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Was this change intentional? We haven't updated to [v23.2](http://android-developers.blogspot.com/2016/02/android-support-library-232.html) of the support library yet, so we're not ready to make this change.